spec/features/provider_context/provider_context_spec.rb

```diff
@@ -72,7 +73,7 @@
         select 'MMT_1', from: 'select_provider'
 
         wait_for_ajax
-
+        click_on 'profile-link'
```

Comment:
Since we now have 2 links that will prompt a user to change their provider context (the badge and the link in the profile link dropdown) should we test both ways to make sure they're working?